src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenIterators.vb

```diff
@@ -1830,5 +1830,28 @@ BC35000: Requested operation is not available because the runtime library functi
                 </expected>)
         End Sub
 
+        <Fact>
+        <WorkItem(220696, "https://devdiv.visualstudio.com/defaultcollection/DevDiv/_workitems#_a=edit&id=220696")>
+        Public Sub IteratorPropertySetterDebugInfo()
+            Dim source =
+<compilation>
+    <file name="a.vb"><![CDATA[
+Imports System
+Imports System.Collections.Generic
+Class C
+    Iterator Property P As IEnumerable(Of Object)
+        Get
+            Yield Nothing
+        End Get
+        Set
+        End Set
+    End Property
+End Class
+]]></file>
+</compilation>
+            Dim compilation = CreateCompilationWithMscorlib(source, options:=TestOptions.DebugDll)
+            compilation.AssertTheseEmitDiagnostics(<expected/>)
+        End Sub
```

Comment:
Can we still iterate over P successfully?
Perhaps make it Yield couple values and print them from a foreach to be sure?